src/engine/Settings.cs

```diff
@@ -478,7 +484,6 @@ private Settings()
     public SettingValue<IReadOnlyList<string>> EnabledMods { get; private set; } = new(new List<string>());
 
     // Computed properties from other settings
-
```

Comment:
Please keep this line, in my commenting style the blank line here has implications regarding the scope of the comment.